src/interactive/Tooltip.jsx

```diff
@@ -22,7 +22,47 @@ class Tooltip extends React.PureComponent {
 		};
 	}
 
+	componentDidMount() {
+		if (this.props.manualToggle) {
+			document.addEventListener('click', this.handleClickOutsideContent);
+		}
+	}
+
+	componentWillUnmount() {
+		if (this.props.manualToggle) {
+			document.removeEventListener('click', this.handleClickOutsideContent);
+		}
+	}
+
+	static getDerivedStateFromProps(nextProps, prevState) {
+		if (nextProps.manualToggle && prevState.isActive !== nextProps.isActive) {
+			return { isActive: nextProps.isActive };
+		}
+		return prevState;
```

Comment:
Is this the same as `return null` ?

Reply:
yes, but this way imo is more explicit of what it is actually doing 😄

Reply:
No doubt, cool with me 🗻 .  Just wanted to be sure haha

Reply:
Interesting. I disagree though, "Returning null is slightly more performant (since React won't have to merge the state with itself). It also more clearly signals the intent of not updating anything." See here and comments related: https://github.com/reactjs/rfcs/issues/26#issuecomment-395407230